doc/telescope.txt

```diff
@@ -116,22 +116,6 @@ builtin.live_grep()                                      *builtin.live_grep()*
 
 Actions functions that are useful for people creating their own mappings.
 
-actions.move_selection_next({prompt_bufnr})    *actions.move_selection_next()*
```

Comment:
hmm, I wonder why this part got deleted as well, was about to merge but I noticed this. I don't want to _accidentally_ delete the existing docs :laughing:

Reply:
This is the max result for a query bug, that is in treesitter core. See https://github.com/nvim-treesitter/nvim-treesitter/issues/963 for all more information. Its super annoying.

Don't know how to proceed right now here.

Reply:
Ooh good catch. I didn't even notice that. @Conni2461 is there any way to *manually* add to the docs for the time being? I'm not sure if that's been tried yet with other PRs, or if it'll still break the docgen action.